dpe/src/dpe_instance.rs

```diff
@@ -468,6 +474,15 @@ impl DpeInstance {
             }
         }
 
+        // Add allow ca to hash
+        if cfi_launder(allow_ca) {
+            hasher.update(true.as_bytes())?;
+        }
+        // Add allow x509 to hash
+        if cfi_launder(allow_x509) {
+            hasher.update(true.as_bytes())?;
+        }
```

Comment:
Instead of doing this conditionally, you could probably always call

```rust
hasher.update(allow_ca.as_bytes());
hasher.update(allow_x509.as_bytes());
```

Reply:
Done